tests/e2e-leg-10/sandbox-basic/59-wait-for-steady-state.yaml

```diff
@@ -15,3 +15,4 @@ apiVersion: kuttl.dev/v1beta1
 kind: TestStep
 commands:
   - command: bash -c "../../../scripts/wait-for-verticadb-steady-state.sh -n verticadb-operator -t 360 $NAMESPACE"
+  - command: sleep 120
```

Comment:
Add this latency to stabilize the cluster before step 60 starts.

Reply:
You don't need to add this sleep. It will take the test longer to complete. **If** this usually fails due to insufficient timeout, then just increase the timeout in the script call above.

Reply:
This is different from timeout. The timeout is the maximum wait time for the whole test step to finish. This sleep call gives the cluster some time to stabilize its state before the next step starts. It makes the test run longer but it makes the test pass.

Reply:
The script on line 17 waits for the operator to be steady(gives the cluster some time to stabilize its state before the next step) meaning there is no error and nothing going on. There is no benefit in adding another wait after. If this step fails and more time is needed, you should increase the time passed as argument to the script.

Reply:
This is from that wait-for-verticadb-steady-state.sh

`timeout $TIMEOUT bash -c -- "while ! $LOG_CMD | \
    grep $WEBHOOK_FILTER | \
    grep $DEPRECATION_FILTER | \
    grep $VDB_FILTER | \
    tail -1 | grep --quiet '\"result\": {\"Requeue\":false,\"RequeueAfter\":0}, \"err\": null'; do sleep 1; done" &
pid=$!
wait $pid
`

This is from "man timeout":
`timeout - run a command with a time limit`

If the script runs longer than the $TIMEOUT, an error will be reported. 
If the script finishes within the $TIMEOUT, the next step starts right away.
What I want to achieve is to add a latency between the two steps to make sure the latter
step is not impacted by the previous step. Increasing the timeout will not achieve that.